handler.js

```diff
@@ -67,16 +78,37 @@ module.exports.start_build = (event, context, callback) => {
     } else if (pullRequest.state != "open") {
       callback("Pull request is not open");
     } else {
-      console.log("Cleared tests, this is a buildable event:", event)
+      console.log("Cleared tests, this is a buildable event:", event);
       response.pull_request = pullRequest;
       var head = pullRequest.head;
       var base = pullRequest.base;
       var repo = base.repo;
 
       var params = {
-        projectName: process.env.BUILD_PROJECT,
-        sourceVersion: 'pr/' + pullRequest.number
+        projectName: process.env.CB_BUILD_PROJECT
       };
+      if (buildOptions.cbGitRepoEnv && buildOptions.cbGitRefEnv ) {
```

Comment:
The logic around this needs a bit of consideration. A git ref is not _required_ when cloning a repo. The clone will simply use the repo's default branch.

Also, what is the rationale for not nesting this test in the `!buildOptions.cbExternalBuildspec` test? I can't think of any reason to use these params separately from that param.